src/TrackGenerator.cpp

```diff
@@ -866,6 +863,15 @@ void TrackGenerator::initializeTrackFileDirectory() {
 
 
 /**
+ * @brief Returns the type of ray tracing used for segment formation
+ * @return the segmentation type
```

Comment:
Perhaps add a `@details` comment block here which enumerates the possible segment formation types for an amateur like me